integreat_cms/deepl_api/utils.py

```diff
@@ -166,6 +166,17 @@ def deepl_translation(self, request, content_objects, language_slug, form_class)
                     )
                     continue
 
+                if not hix_is_sufficient(source_translation):
+                    messages.error(
+                        request,
+                        _(
+                            "HIX score of {:.2f} is too low for machine translation (minimum required: {})"
+                        ).format(
+                            source_translation.hix_score,
+                            settings.HIX_REQUIRED_FOR_MT,
+                        ),
+                    )
+
                 existing_target_translation = content_object.get_translation(
```

Comment:
```suggestion
                if getattr(source_translation.foreign_object, "hix_ignore", False):
                    messages.error(
                        request,
                        _(
                            'Machine translations are disabled for "{}", because its HIX value is ignored'
                        ).format(
                            source_translation.title,
                        ),
                    )
                    return False

                existing_target_translation = content_object.get_translation(
```
Ah, one thing I forgot: We also have to check, whether the new field `hix_ignore` is set (and disable MT in this case, see #2075)

Reply:
Also in the UI we should probably disable or hide the automatic translation checkbox when the hix ignore button is pressed...